internal/cephfs/nodeserver.go

```diff
@@ -270,6 +272,9 @@ func (ns *NodeServer) NodeStageVolume(
 		}
 	}
 
+	err = ns.healthChecker.StartChecker(req.GetVolumeId(), stagingTargetPath)
```

Comment:
That is not part of this PR. This only addresses the async health-checking. #4125 was started with the extra `data` subdir in mind, and it will be remodelled to extend this PR.